### PR TITLE
194_Update_"Tagging-your-load"_Sections

### PR DESCRIPTION
**Pull request by Dmac26**
﻿## Summary
Addresses Issue # 194
https://app.zenhub.com/workspaces/open-forest-58f8bbe105a35fad2d275a0c/issues/usdaforestservice/usfs-timber-permitting/194

This PR addresses changes requested for the forest info pages in regards to the "tagging your load" section. Four html files and 1 sass file have been changed to address this request.
## This pull request is ready to merge when...
- [x] Feature branch starts with the issue number
- [x] Is connected to its original issue via zenhub 👇
- [x] All tests are passing and meet coverage, linting, and accessibility requirements. And no security vulnerabilities ⚫️(Circle)
- [x] This code has been reviewed by someone other than the original author

### CODE REVIEW

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html [hunk 1]
```diff
@@ -119,47 +119,64 @@ <h3>What to expect</h3>
...
+  <h3>To attach a tag, you'll need to:</h3>
+  <ul>
+    <li>Fold your tag on the dotted lines.</li>
+    <li>Place each tag in a zip-top bag, and close the bag.</li>
+    <li>Nail your tag(s) to a piece of wood in the back of your load so that it is visible to cars behind you.</li>
+  </ul>
+</div>
+
+<!-- Tagging Your Load Block for Region 2 -->
+<div *ngIf="(forest.forestAbbr)=='shoshone' || (forest.forestAbbr)=='GMUG'">
```
Comment (briandavidson) [inline frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html]: I'd prefer `(forest.forestAbbr) ===  'shoshone'` but no biggie, the code works.

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/where-to-find/where-to-find.component.html [hunk 2]
```diff
@@ -77,7 +77,7 @@ <h3>Prohibited areas and restrictions</h3>
 
 <!-- flathead block -->
 <div *ngIf="(forest.forestAbbr)=='flathead'">
```
Comment (briandavidson) [inline frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/where-to-find/where-to-find.component.html]: If you do end up changing the code style above, would you mind fixing this one as well?

**Review by briandavidson — APPROVED** [review]
A couple small things to take a look at but no show stoppers. 👍

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html (hunk 1 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/how-to-cut/how-to-cut.component.html]: @briandavidson - I went ahead and corrected all equality operators to `===`

File: frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/where-to-find/where-to-find.component.html (hunk 2 above)
Comment (Dmac26) [reply frontend/src/app/firewood/forests/firewood-guidelines/forest-guidelines/where-to-find/where-to-find.component.html]: donesky!

**Pull request merged**